crates/sui-json-rpc/src/read_api.rs

```diff
@@ -166,6 +166,31 @@ impl ReadApiServer for ReadApi {
         })
     }
 
+    async fn multi_get_transactions(
+        &self,
+        digests: Vec<TransactionDigest>,
+    ) -> RpcResult<Vec<SuiTransactionResponse>> {
+        let txn_batch = self
```

Comment:
we should ensure there are no duplicates and also limit the number of digest before any processing.

we can use `QUERY_MAX_RESULT_LIMIT` for now